requirements.dev.txt

```diff
@@ -2,4 +2,4 @@
 pytest
 pytest-django
 Faker
-factory-boy
```

Comment:
Is there a reason why we removed factory-boy from here?

Reply:
we decide not to use it in this PR 
you can add it later if it will be needed